DBNetworkStack+Sourcing/ResourceDataProvider.swift

```diff
@@ -122,12 +108,7 @@ public class ResourceDataProvider<Object>: ArrayDataProviding {
     
     func loadDidSucess(with newFetchedData: Array<Object>) {
```

Comment:
Cause there are no 'old fetched data' change 'newFetchedData' to just 'fetchedData'